tools/ncexists/ncexists.c

```diff
@@ -137,6 +137,8 @@ int main (int argc, char **argv)
 	 }
 
 
+   status = nc_sync(ncid);
+       if (status != NC_NOERR) handle_error(status);
```

Comment:
Are we actually manipulating the contents of the file in the ncexists application?  If we aren't, then all opens should be of the nowrite variety and there will be no need to sync prior to close.

Reply:
No it's not doing any writing it uses NC_NOWRITE for the open call. I removed the sync call and the nc_open call was just passing in 0 so i updated it to use the macro name from netcdf as well.